### PR TITLE
Add user functionality manager feature.

### PR DESCRIPTION
**Pull request by ChamathNS**
Merge after https://github.com/wso2/carbon-identity-framework/pull/2972

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1861,6 +1866,16 @@
                 <artifactId>org.wso2.carbon.identity.template.mgt.feature</artifactId>
                 <version>${carbon.identity.framework.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.identity.framework</groupId>
+                <artifactId>org.wso2.carbon.identity.user.functionality.mgt.server.feature</artifactId>
+                <version>${project.version}</version>
```
Comment (pamodaaw) [inline pom.xml:1872]: Shouldn't this be the version of the carbon-identity-framework?

File: pom.xml [hunk 2]
```diff
@@ -1861,6 +1866,16 @@
...
+            </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.identity.framework</groupId>
+                <artifactId>org.wso2.carbon.identity.user.functionality.mgt.feature</artifactId>
+                <version>${project.version}</version>
```
Comment (pamodaaw) [inline pom.xml:1877]: Shouldn't this be the version of the carbon-identity-framework?

File: pom.xml [hunk 3]
```diff
@@ -1871,6 +1876,16 @@
                 <artifactId>org.wso2.carbon.identity.cors.mgt.server.feature</artifactId>
                 <version>${carbon.identity.framework.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.identity.framework</groupId>
+                <artifactId>org.wso2.carbon.identity.user.functionality.mgt.server.feature</artifactId>
+                <version>${carbon.identity.framework.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.identity.framework</groupId>
```
Comment (mefarazath) [inline pom.xml:1885]: Do we need this as well or having the server feature is sufficient?

Comment (ChamathNS) [reply pom.xml]: Resolved with https://github.com/wso2/product-is/pull/8502/commits/4193975e6aed6857f6efb4fe502d88b91e25bd72.

**Review by mefarazath — APPROVED** [review]

**Pull request merged**